dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentTypeFactoryImplTest.java

```diff
@@ -189,6 +196,31 @@ public void testAddingContentTypes() throws Exception {
 		assertThat("contenttypes are added", count == count2 - runs);
 	}
 
+	@Test
```

Comment:
Doc Test 
```
/**
* Method to test: include this in case the method to test does not belong to the class this test-class is testing 
* Given Scenario: Scenario under test
* ExpectedResult: Expected result that the assertion is checking against
*
*/
```

Reply:
Added.